tests/automation/setup/beforeEach.ts

```diff
@@ -13,7 +13,7 @@ const getDirectoriesOfSessionDataPath = (source: string) =>
     })
     .filter((n) => n.includes(`${NODE_ENV}-${MULTI_PREFIX}`));
 
-const alreadyCleaned = false;
+const alreadyCleaned = true;
```

Comment:
not sure if we should maybe rely on an environment variable for that? What do you think?

Reply:
I think this might have been a quick fix for playwright not liking concurrent windows opening at once. Like check the folder if it's clean through the ENV?